app/models/insured_eligible_for_benefit_rule.rb

```diff
@@ -51,7 +51,9 @@ def satisfied?
           true && eligible
         else
           if "#{element}" == "active_consumer"
-             @errors << ["eligibility failed on market kind"]
+            @errors << ["eligibility failed on market kind"]
+          elsif FinancialAssistanceRegistry[:consumer_validations].enabled?
```

Comment:
Ok, so this is in the main app.

The consumer_validations setting needs to be held in the file: `system/config/templates/features/aca_individual_market/financial_assistance.yml`.

Because its in the main app, we should be referencing it with `EnrollRegistry` and for consistency with `EnrollRegistry.feature_enabled?(:consumer_validations)`. It's simply called `FinancialAssistanceRegistry` when the `resource_registry.rb` initialized in the file here: https://github.com/ideacrew/enroll/blob/trunk/components/financial_assistance/config/initializers/resource_registry.rb